action.yml

```diff
@@ -2,30 +2,36 @@ name: "Run remark-lint with reviewdog"
 description: "🐶 Run remark-lint with reviewdog on pull requests to improve code review experience."
 author: "prologic"
 inputs:
-  github_token:
-    description: "GITHUB_TOKEN."
-    required: true
   workdir:
-    description: "Working directory relative to the root directory."
+    description: "Working directory relative to the root directory. Defaults to '.'."
     required: false
     default: "."
+  fail_on_error:
+    description: |
+      Exit code for reviewdog when errors are found [true, false]. Defaults to 'false'.
+    required: false
+    default: "false"
   remark_flags:
-    description: "Additional flags for remark-lint"
+    description: "Additional remark-lint flags."
     required: false
     default: ""
+  # Reviewdog related inputs
+  github_token:
+    description: "The automatically created secret github action token."
+    required: true
+    default: ${{ github.token }}
   tool_name:
-    description: "Tool name to use for reviewdog reporter"
+    description: "Tool name to use for reviewdog reporter. Defaults to 'black-format'."
     required: false
     default: "remark-lint"
   level:
-    description: "Report level for reviewdog [info,warning,error]"
+    description: "Report level for reviewdog [info, warning, error]. Defaults to 'error'."
     required: false
     default: "error"
   reporter:
     description: |
-      Reporter of reviewdog command [github-pr-check,github-pr-review,github-check].
-      Default is github-pr-check.
-      github-pr-review can use Markdown and add a link to rule page in reviewdog reports.
+      Reporter of reviewdog command [github-pr-check, github-pr-review, github-check].
+      Defaults to 'github-pr-check'.
```

Comment:
I would keep the same format of description everywhere.
So, either we write in two lines.

```
description: |
<Desc>
<Default>
```

or
```
description:  <Desc> <Default>
```

Reply:
@limakz I think your right. Neverteless do you think we really need to explicitly state the default inside the description tag. The default is already stated in both the README.md and can be found inside the `defaults` tag.

Reply:
@limakzi I went ahead and made these changes see cdae7e2. I can revert if you think adding it is still benificial.